doc/files.md

```diff
@@ -1,36 +1,37 @@
+bitcoin.conf        | contains configuration settings for bitcoind or bitcoin-qt                                                                 
+bitcoind.pid        | stores the process id of bitcoind while running                                                                            
+blocks/blk000??.dat | block data (custom, 128 MiB per file); since 0.8.0                                                                         
+blocks/rev000??.dat | block undo data (custom); since 0.8.0 (format changed since pre-0.8)                                                       
+blocks/index/*      | block index (LevelDB); since 0.8.0                                                                                         
+chainstate/*        | blockchain state database (LevelDB); since 0.8.0                                                                           
+database/*          | BDB database environment; only used for wallet since 0.8.0; moved to wallets/ directory on new installs since 0.16.0       
+db.log              | wallet database log file; moved to wallets/ directory on new installs since 0.16.0                                         
+debug.log           | contains debug information and general logging generated by bitcoind or bitcoin-qt                                         
+fee_estimates.dat   | stores statistics used to estimate minimum transaction fees and priorities required for confirmation; since 0.10.0         
+indexes/txindex/*   | optional transaction index database (LevelDB); since 0.17.0                                                                
+mempool.dat         | dump of the mempool's transactions; since 0.14.0                                                                           
+peers.dat           | peer IP address database (custom format); since 0.7.0                                                                      
+wallet.dat          | personal wallet (BDB) with keys and transactions; moved to wallets/ directory on new installs since 0.16.0                 
+wallets/database/*  | BDB database environment; used for wallets since 0.16.0                                                                    
+wallets/db.log      | wallet database log file; since 0.16.0                                                                                     
+wallets/wallet.dat  | personal wallet (BDB) with keys and transactions; since 0.16.0                                                             
+.cookie             | session RPC authentication cookie (written at start when cookie authentication is used, deleted on shutdown): since 0.12.0 
+onion_private_key   | cached Tor hidden service private key for -listenonion: since 0.12.0                                                       
+guisettings.ini.bak | backup of former GUI settings after -resetguisettings is used                                                              

```

Comment:
Keep the formatting unchanged - preserve backticks :-)

Reply:
Ok just was confused by the travis errors